Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:focal
+FROM ubuntu:jammy
```

Comment:
Is there a reason not to use `ubuntu:noble`, the latest LTS?

Reply:
Yes, I was worried about breaking u.com, but it has since been updated. I'll run tests with noble for ubuntu.com, canonical.com and snapcraft.io

Reply:
I've just tested this with `noble` on ubuntu.com and there are quite a few breaking changes, which I think deserve closer attention and perhaps another PR.
For now, I think `jammy` is alright, also considering that most sites are using `jammy`.